### PR TITLE
Elastic search krkn-lib integration

### PR DESCRIPTION
**Pull request by tsebastiani**
Collects:

- Telemetry structure
- metrics
- alerts

### CODE REVIEW

File: config/config.yaml [hunk 1]
```diff
@@ -53,12 +53,28 @@ cerberus:
...
+    enable_elastic: True
+    collect_metrics: True
+    collect_alerts: True
+    verify_certs: False
+    elastic_url: ""                                         # To track results in elasticsearch, give url to server here; will post telemetry details when url and index not blank
+    elastic_port: 32766
+    elastic_index: ""                                       # Elastic search index pattern to post results to
+    username: "elastic"
+    password: "test"
+    metrics_index: "krkn-metrics"
```
Comment (paigerube14) [inline config/config.yaml:74]: Here are we pushign the metrics, alerts and telemetry all separately?

**Comment by paigerube14** [comment]
Got this import error after I installed  the new requirements file

```
% python run_kraken.py 
Traceback (most recent call last):
  File "/Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/run_kraken.py", line 13, in <module>
    from krkn_lib.elastic.krkn_elastic import KrknElastic
  File "/Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/venv3/lib/python3.10/site-packages/krkn_lib/elastic/__init__.py", line 1, in <module>
    from .krkn_elastic import *
  File "/Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/venv3/lib/python3.10/site-packages/krkn_lib/elastic/krkn_elastic.py", line 10, in <module>
    from elasticsearch_dsl import Search
ModuleNotFoundError: No module named 'elasticsearch_dsl'
```

**Comment by paigerube14** [comment]
Once I installed the requirement I hit 

```
2024-07-19 13:37:59,757 [INFO] 
Traceback (most recent call last):
  File "/Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/run_kraken.py", line 579, in <module>
    main(options.cfg)
  File "/Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/run_kraken.py", line 445, in main
    telemetry_k8s.collect_cluster_metadata(chaos_telemetry)
  File "/Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/venv3/lib/python3.10/site-packages/krkn_lib/telemetry/k8s/krkn_telemetry_kubernetes.py", line 65, in collect_cluster_metadata
    node_infos, taints = self.kubecli.get_nodes_infos()
  File "/Users/prubenda/PycharmProjects/pplRepos/tsebastiani/krkn/venv3/lib/python3.10/site-packages/krkn_lib/k8s/krkn_kubernetes.py", line 2058, in get_nodes_infos
    taint = Taint(
TypeError: Taint.__init__() got an unexpected keyword argument 'node_name'
```

File: config/config.yaml [hunk 2]
```diff
@@ -53,12 +53,28 @@ cerberus:
...
     repo: "https://github.com/cloud-bulldozer/performance-dashboards.git"
-    prometheus_url:                                      # The prometheus url/route is automatically obtained in case of OpenShift, please set it when the distribution is Kubernetes.
+    prometheus_url: ''                                      # The prometheus url/route is automatically obtained in case of OpenShift, please set it when the distribution is Kubernetes.
     prometheus_bearer_token:                              # The bearer token is automatically obtained in case of OpenShift, please set it when the distribution is Kubernetes. This is needed to authenticate with prometheus.
     uuid:                                                 # uuid for the run is generated by default if not set
-    enable_alerts: False                                  # Runs the queries specified in the alert profile and displays the info or exits 1 when severity=error
+    enable_alerts: True                                  # Runs the queries specified in the alert profile and displays the info or exits 1 when severity=error
```
Comment (chaitanyaenr) [inline config/config.yaml:59]: We might want to disable alerts check and metrics collection by default. Thoughts?

File: config/config.yaml (hunk 1 above)
Comment (paigerube14) [reply config/config.yaml:76]: can telemetry_index and elastic_index be one setting?

**Review by paigerube14 — APPROVED** [review]
/lgtm

**Review by chaitanyaenr — APPROVED** [review]
LGTM

**Pull request merged**